cfgov/v1/auth_forms.py

```diff
@@ -122,4 +122,11 @@ def confirm_login_allowed(self, user):
             raise ValidationError("This account is temporarily locked; please try later or <a href='/admin/password_reset/' style='color:white;font-weight:bold'>reset your password</a>")
 
 
+class UserCreationForm(wagtailforms.UserCreationForm):
+    def save(self, commit=True):
+        user = super(UserCreationForm, self).save(commit=commit)
 
+        if commit:
+            send_password_reset_email(user.email)
```

Comment:
It would nice to be able to put a `try/except` around this, and then log a warning if the email send fails (but still return to the Wagtail admin). Is there an existing convention about logging this kind of non-critical error?